### PR TITLE
Use chrome HTML to fix failing ESI tag mutations.

### PR DESCRIPTION
**Pull request by Hyperkid123**
There are ongoing issues caused by either `webpack-dev-server` itself or one of its dependencies with mutating the ESI tags inside the HTML templates with the actual content.

We do not know the cause of the issue nor how to fix it. This PR uses a chrome HTML template directly in local env. As this is approach will be used on products in the future, we can fast track it in this repository to unblock the development.

### CODE REVIEW

File: config/dev.webpack.config.js [hunk 1]
```diff
@@ -19,7 +20,6 @@ const { config: webpackConfig, plugins } = config({
             changeOrigin: true,
             autoRewrite: true,
             ws: true,
-            ...['prod', 'stage'].includes(process.env.ENVIRONMENT) && { agent: new HttpsProxyAgent('http://squid.corp.redhat.com:3128') }
```
Comment (karelhala) [inline config/dev.webpack.config.js:22]: Isn't this still required when user runs custom local API?

Comment (Hyperkid123) [reply config/dev.webpack.config.js:22]: Nope. That is already a part of the webpack proxy

Comment (Hyperkid123) [reply config/dev.webpack.config.js:22]: https://github.com/RedHatInsights/frontend-components/blob/master/packages/config-utils/proxy.js#L58

Comment (karelhala) [reply config/dev.webpack.config.js:22]: Yeah, I know it's part of the general proxy. But previously there were errors when this config was missing from the customProxy as well. Is there any benefit of removing it?

Comment (Hyperkid123) [reply config/dev.webpack.config.js:22]: I don't have the local API setup. I would need someone who has to test it. I think having the proxy agent active twice might have some unwanted issues.

Comment (karelhala) [reply config/dev.webpack.config.js:22]: Sure thing, let's remove it. If someone points to it that it won't work we'll re-add it.

**Review by karelhala — APPROVED** [review]

**Pull request merged**